inspector/src/components/graph/canvasGraphService.ts

```diff
@@ -416,9 +459,131 @@ export class CanvasGraphService {
      */
     private _removeEventListeners(canvas: HTMLCanvasElement) {
         canvas.removeEventListener("wheel", this._handleZoom);
+        canvas.removeEventListener("mousemove", this._handleDataHover);
         canvas.removeEventListener("mousedown", this._handlePanStart);
+        canvas.removeEventListener("mouseleave", this._handleStopHover);
         canvas.ownerDocument.removeEventListener("mouseup", this._handlePanStop);
     }
+    
+    /**
+     * Handles what to do when we are hovering over the canvas and not panning.
+     * 
+     * @param event A reference to the event to be handled. 
+     */
+    private _handleDataHover = (event: MouseEvent) => {
+        if (this._panPosition) {
+            // we don't want to do anything if we are in the middle of panning
+            return;
+        }
+
+        this._hoverPosition = event.clientX;

```

Comment:
Should this not account for the canvas not being fully at 0 on the left ? (It might already be the case but just guessing)

Reply:
it does already!